minio/go/src/cloud/benchflow/commons/minio/minio.go

```diff
@@ -53,11 +31,29 @@ func callMinioClient(fileName string, minioHost string, minioKey string) {
 		fmt.Println("Result: " + out.String())
 }
 
-func GenerateKey(fileName string) string {
-	// TODO: Use the hash library to generate a hash
-	trialId := os.Getenv("TRIAL_ID")
-	t := strings.Split(trialId, "_")
-	return ("hash/"+t[0]+"/"+t[1]+"/"+os.Getenv("CONTAINER_NAME")+"/"+os.Getenv("COLLECTOR_NAME")+"/"+os.Getenv("DATA_NAME")+"/"+fileName)
+func sendToMinio(fileName string, minioHost string, minioKey string, accessKey string, secretAccessKey string) {
+	minioClient, err := minio.New(minioHost, accessKey, secretAccessKey, true)
+	if err != nil {
+		fmt.Println(err)
+	    return
+	    }
+	_, err = minioClient.FPutObject("runs", minioKey, fileName, "application/gzip")
```

Comment:
You set `application/gzip`, then the function's name should refer to gzip and call another generic function `sendToMinio` accepting the MIME type as parameter